### ISSUE TITLE
Extra whitespace

### ISSUE THREAD

**Issue by madstap** [kkinnear/zprint#59]

**Comment by madstap** [comment]
There seems to be some extra whitespace here? Or maybe I'm asking for a configuration option?

```clj

(require
 '[clojure.string :as str]
 '[zprint.core :as zp])

(defn f [s]
  (-> s
      (zp/zprint-file-str nil {:style [:map-nl]})
      (str/replace \space \*)
      (println)))

(comment

  (f "(let [foo {:some-really-really-long-name/foo.bar :some-really-really-long-name/baz.quux :some-really-really-long-name/xxxxxxxxxxxxxxxxxxxxxx :some-really-long-name/yyyyyyyyyyyyyyy}])")

  (f "{:some-really-really-long-name/foo.bar.baz-quux :some-really-really-long-name/baz.quux.foo-bar :some-really-really-really-long-name/xxxxxxxxxxxxxxxxxxxxxx :some-really-really-really-long-name/yyyyyyyyyyyyyyy}")

  )
```

Evaluating the forms prints this:

```
(let*[foo
******{:some-really-really-long-name/foo.bar
*******:some-really-really-long-name/baz.quux,
*******
*******:some-really-really-long-name/xxxxxxxxxxxxxxxxxxxxxx
*******:some-really-long-name/yyyyyyyyyyyyyyy}])

{:some-really-really-long-name/foo.bar.baz-quux
*:some-really-really-long-name/baz.quux.foo-bar,
*
*:some-really-really-really-long-name/xxxxxxxxxxxxxxxxxxxxxx
*:some-really-really-really-long-name/yyyyyyyyyyyyyyy}
```

While I expected it to print this:
```
(let*[foo
******{:some-really-really-long-name/foo.bar
*******:some-really-really-long-name/baz.quux,

*******:some-really-really-long-name/xxxxxxxxxxxxxxxxxxxxxx
*******:some-really-long-name/yyyyyyyyyyyyyyy}])

{:some-really-really-long-name/foo.bar.baz-quux
*:some-really-really-long-name/baz.quux.foo-bar,

*:some-really-really-really-long-name/xxxxxxxxxxxxxxxxxxxxxx
*:some-really-really-really-long-name/yyyyyyyyyyyyyyy}
```

**Comment by kkinnear** [comment]
Fixed in 0.4.10.
